config/base/env/config

```diff
@@ -6,13 +6,7 @@ DB_NAME=tekton-results
 DB_SSLMODE=disable
 DB_SSLROOTCERT=
 DB_ENABLE_AUTO_MIGRATION=true
-DB_MAX_IDLE_CONNECTIONS=10
```

Comment:
I'm assuming you did this @sayan-biswas because I didn't need to set these defaults when I added the tuning options ?  If so, thanks for the correction.

Reply:
Or was this accidental?  ...If it was, in hindsight, I does probably make sense to not set these, even to the defaults. (though 10 does not line up with the default as described in the doc.

Users can always add these to their config map.